.github/workflows/build-and-deploy-master-proxy.yml

```diff
@@ -148,6 +150,15 @@ jobs:
         with:
           task-definition: ${{ steps.render-task-def.outputs.task-definition }}
 
+      - name: Find current revision
+        id: current-revision
+        run: |
+          current_revision=$(aws ecs describe-task-definition --task-definition "${{ steps.get-task-def.outputs.TASK_DEFINITION_ARN }}" --query 'taskDefinition.revision' --output text)
+          echo "Current ECS task revision: $current_revision"
+          echo "REVISION=$current_revision" >> "$GITHUB_OUTPUT"
+
       - name: Restart Luovutuspalvelu proxy
         run: |
-          aws ecs update-service --cluster "${{ env.CLUSTER }}" --service "${{ env.TASK_FAMILY }}" --force-new-deployment --output text
+          current_revision=${{ steps.current-revision.outputs.REVISION }}
+          new_revision=$((current_revision + 1))
```

Comment:
Hmm, vähän tämä näyttää kyllä oudolta.

Vilkaisin https://github.com/aws-actions/amazon-ecs-deploy-task-definition -dokumentaatiota, jos oikein ymmärsin, niin siellä esimerkissä tuo aiempi aws-actions/amazon-ecs-deploy-task-definition hoitaa myös palvelun restarttaamisen uudella task definitionilla. Miksi meillä pitää tehdä restart erikseen?

Reply:
Pelkästään tolla komennolla syntyy kyllä uus revision task definitionista mutta ei sitä käynnistetä 🤷

Reply:
Toi on kyllä mahdollisesti joku konfiguraatioasia kun vaan ymmärtäis vähän enemmän tosta ecs:stä'

Reply:
Esim. tästä vois olla kyse:

minimumHealthyPercent

If a service is using the rolling update (ECS) deployment type, the minimumHealthyPercent represents a lower limit on the number of your service's tasks that must remain in the RUNNING state during a deployment, as a percentage of the desiredCount (rounded up to the nearest integer). This parameter enables you to deploy without using additional cluster capacity. For example, if your service has a desiredCount of four tasks and a minimumHealthyPercent of 50%, the service scheduler may stop two existing tasks to free up cluster capacity before starting two new tasks.

Reply:
Ahaa, siellä on tuollaista monimutkaisuutta, ei tuota ole varmaan oikein optimoitu tapauksiin, joissa on vain yksi kontti, jonka voi silloin tällöin uudelleenkäynnistää.

Reply:
No nyt selvis että tää on luultavasti feilannu koko ajan hiljaisesti, tarvittavat inframuutokset on matkalla deviin